packages/manager-components/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@ovhcloud/manager-components",
+  "private": true,
+  "version": "1.0.0",
+  "main": "src/components/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager-components"
+  },
+  "scripts": {
+    "start": "storybook dev -p 6006",
+    "dev": "tsc",
+    "build": "echo manager-components",
+    "prettier": "prettier --write \"src/**/*.{ts,tsx,js,mdx}\"",
+    "test": "jest --coverage -u",
+    "test:cov": "vitest run --coverage",
+    "test:e2e": "tsc && npx playwright test --ui --debug"
+  },
+  "dependencies": {
+    "babel-preset-react-app": "^10.0.1",
+    "@ovhcloud/ods-common-core": "^16.6.0",
+    "@ovhcloud/ods-theme-blue-jeans": "^16.6.0",
+    "@ovhcloud/ods-common-stencil": "^16.6.0",
+    "@ovhcloud/ods-common-testing": "^16.6.0",
+    "@ovhcloud/ods-common-theming": "^16.6.0",
+    "@ovhcloud/ods-components": "^16.6.0",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "i18next": "^20.4.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-i18next": "^11.18.1",
+    "tailwindcss": "3.3.3"
+  },
+  "devDependencies": {
+    "@babel/core": "7.22.10",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@ovh-ux/manager-tailwind-config": "*",
+    "@storybook/addon-essentials": "7.5.3",
+    "@storybook/addon-interactions": "7.5.3",
+    "@storybook/addon-links": "7.5.3",
+    "@storybook/addon-styling": "1.3.6",
+    "@storybook/blocks": "7.5.3",
+    "@storybook/react": "7.5.3",
+    "@storybook/react-vite": "7.5.3",
+    "@storybook/testing-library": "0.2.0",
+    "@testing-library/react": "14.0.0",
+    "@types/jest": "^29.5.5",
+    "@types/node": "20.4.9",
+    "@types/react": "18.2.20",
+    "@types/react-dom": "18.2.7",
+    "@typescript-eslint/eslint-plugin": "5.60.0",
+    "@vitejs/plugin-react": "4.0.4",
+    "@vitest/coverage-v8": "0.34.1",
+    "autoprefixer": "10.4.14",
+    "babel-loader": "9.1.3",
+    "eslint-config-prettier": "9.0.0",
+    "eslint-config-standard-with-typescript": "37.0.0",
+    "eslint-plugin-import": "2.28.0",
+    "eslint-plugin-n": "16.0.1",
+    "eslint-plugin-promise": "6.1.1",
+    "eslint-plugin-react": "7.33.1",
+    "eslint-plugin-storybook": "0.6.13",
+    "file-loader": "^6.2.0",
+    "husky": "8.0.3",
+    "i18next": "^20.4.0",
```

Comment:
This is duplicate. Already present in `dependencies`

Reply:
Ok I delete it